debugging/netcoreapp/outputManager.ts

```diff
@@ -0,0 +1,82 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import * as vscode from 'vscode';
```

Comment:
At a high level, what does this do?

Reply:
It wraps output going to a VS Code `OutputChannel`, allowing it to be grouped into operations, i.e. with a starting message, ending message, error message, where the operation content is prefixed.  It allows a simple visualization of a hierarchy of operations.  This is often needed when performing Docker operations, where it's desired to display the raw output of the operation but ensure it's distinct from the output of the extension itself.

For example:

```
Starting Docker build...
> Sending build context to Docker daemon   2.47MB
> Step 1/16 : FROM microsoft/dotnet:2.1-aspnetcore-runtime AS base
> ---> 2967770cac05
> Step 2/16 : WORKDIR /app
> ---> Using cache
> ---> 7c9558751bf3
> ...
Docker build complete.
```